db/dataset_eval.py

```diff
@@ -296,6 +296,28 @@ def _create_job(connection, dataset_id, normalize, eval_location, filter_type=No
     return job_id
 
 
+def get_user_pending_jobs(user_api_key):
```

Comment:
I think this would make more sense to be a method which gets pending jobs for a _user_, not an ID, so you could call it `get_pending_jobs_for_user`